### PR TITLE
Add Hazy theme

### PR DESCRIPTION
**Pull request by fitzthomas18**
adds [hazy](https://github.com/Astromations/Hazy)

### CODE REVIEW

**Comment by gigamonster256** [comment]
I’m sure adding this to the THEMES.md doc would be appreciated

**Comment by fitzthomas18** [comment]
my apologies, it is added now.

File: pkgs/themes.nix [hunk 1]
```diff
@@ -223,6 +223,21 @@
       '';
     };
 
+    hazy = {
+      name = "Hazy";
+      src = sources.hazySrc;
+      appendName = false;
+      injectCss = true;
+      replaceColors = true;
+      overwriteAssets = false;
```
Comment (Gerg-L) [inline pkgs/themes.nix]:
```suggestion
```
appendName was removed, the other three are set to the defaults and not needed

**Review by Gerg-L — CHANGES_REQUESTED** [review]
This then I'll merge

**Comment by Gerg-L** [comment]
Thanks @gigamonster256

**Comment by Gerg-L** [comment]
Thanks for contributing!

**Pull request merged**